### PR TITLE
Improves Travis Efficiency by trimming some Builds  #148105193

### PR DESCRIPTION
**Pull request by daumie**
### What does this PR do?
Improves Travis CI Efficiency

### Description of tasks completed.
Have Travis CI build time decreased by removing some versions
of python and development database

### What are the relevant pivotal tracker stories?
 #148105193

### CODE REVIEW

File: wger/settings_global.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
 SITE_ROOT = os.path.realpath(os.path.dirname(__file__))
 # DATABASES[‘default’] = dj_database_url.config()
 
-DATABASES = {'default': dj_database_url.config(default=os.environ["DATABASE_URL"])}
+# DATABASES = {'default': dj_database_url.config(default=os.environ["DATABASE_URL"])}
```
Comment (egichuri) [inline wger/settings_global.py]: we need this db configuration for the sake of heroku

File: .travis.yml [hunk 2]
```diff
@@ -42,12 +39,11 @@ install:
   - nvm use $TRAVIS_NODE_VERSION
 
   # Install requirements
-  - pip install -r requirements_devel.txt
+  - pip install -r requirements.txt
```
Comment (egichuri) [inline .travis.yml]: any reason why this was changed from `requirements_devel.txt`? It's importing `requirements.txt` so just ensure no packages are repeated in both.

**Review by egichuri — CHANGES_REQUESTED** [review]

File: wger/settings_global.py (hunk 1 above)
Comment (daumie) [reply wger/settings_global.py]: I will have this corrected, this means the build failure has something to do with this. Any ideas of possible workarounds?

**Review by daumie — COMMENTED** [review]
I have removed repeated packages from **requirements.txt**. 
Re-added the **heroku db** configuration.
I will need help fixing the failing

**Comment by egichuri** [comment]
This has already been merged

**Pull request merged**